openapi/sled-agent.json

```diff
@@ -1917,7 +1917,8 @@
                 "description": "External DNS servers Nexus can use to resolve external hosts.",
                 "type": "array",
                 "items": {
-                  "type": "string"
+                  "type": "string",
```

Comment:
Just checking: this is not a backwards-incompatible change, is it?

Reply:
You're right, it wouldn't just work on an existing install without updating the corresponding nexus service config on whatever sled `/pool/int/*/config/services.toml`.

Reply:
Oh, that's worse than I realized.  Is this...okay?

Reply:
I had assumed we're still at the point of requiring a clean slate on mupdate? (especially given in-flight work like trust quorum, early networking, etc)

Reply:
Yes, for a few more days, but probably not by the end of next week